### PR TITLE
refactor:renamed video channels to video

### LINKED ISSUE #7183: Rename Admin "video-channels" to "Video"
Please simplify the tab "video-channels" to be "Video" in the admin section of the platform.

### PR DESCRIPTION
**Pull request by tanay123456789**
Fixes #7183 

#### Changes proposed in this pull request:
Renamed video channels to video

#### Checklist

- [x] I have read the [Contribution & Best practices Guide](https://blog.fossasia.org/open-source-developer-guide-and-best-practices-at-fossasia).
- [x] My branch is up-to-date with the Upstream `development` branch.
- [x] The acceptance, integration, unit tests and linter pass locally with my changes 
- [x] I have added tests that prove my fix is effective or that my feature works
- [x] I have added necessary documentation (if appropriate)

### CODE REVIEW

File: app/components/forms/events/view/videoroom-form.hbs [hunk 1]
```diff
@@ -2,7 +2,7 @@
   <div class="ui grid">
     <div class="ui left aligned sixteen wide mobile eight wide computer column">
       <h3 class="ui header">
-        {{t 'Video Channel'}}
```
Comment (MeghalBisht) [inline app/components/forms/events/view/videoroom-form.hbs:5]: The issue is about the admin section

Comment (tanay123456789) [reply app/components/forms/events/view/videoroom-form.hbs:5]: ok working on it

Comment (tanay123456789) [reply app/components/forms/events/view/videoroom-form.hbs:5]: @MeghalBisht

Comment (MeghalBisht) [reply app/components/forms/events/view/videoroom-form.hbs:5]: No, this is not the file to be changed. The issue talks about the admin section.

Comment (tanay123456789) [reply app/components/forms/events/view/videoroom-form.hbs:5]: @MeghalBisht now see it

Comment (tanay123456789) [reply app/components/forms/events/view/videoroom-form.hbs:5]: @MeghalBisht if there is anything wrong in my approach then pls ma'am guide me.

Comment (MeghalBisht) [reply app/components/forms/events/view/videoroom-form.hbs:5]: I dont think you are testing the changes. Still wrong. Run the project and use ember inspector to see the file to be changed

**Comment by MeghalBisht** [comment]
Inactive

**Pull request closed without merging**